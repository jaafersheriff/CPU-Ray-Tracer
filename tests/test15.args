firsthit spheres.pov 640 480 135 340
