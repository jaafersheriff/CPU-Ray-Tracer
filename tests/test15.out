Pixel: [135, 340] Ray: {0 0 14} -> {-0.3521 0.1918 -0.9161}
T = 14.24
Object Type: Sphere
Color: 0.12 0.34 0.56
